tatu_re/model/train.py

```diff
@@ -0,0 +1,29 @@
+from tatu_re.model.data_processing import calculate_target
+from tatu_re.model.linear_regression import linear_regression_pipeline
+from tatu_re.utils import get_data
+
+from datetime import datetime
+import pathlib
+import pickle
+
+def train_model(save=True):
```

Comment:
Maybe it's good to add the begin and end dates as parameters to this function

Reply:
Yes, I created something nice to execute this function. But I don't know yet how to use parameters.

Basically you run `pip install -e .` in the directory, this installs the dependencies but also allows to run a script that calls train-model. Which executes this code.